settings/ui/components/SettingsApp.vue

```diff
@@ -27,7 +27,7 @@
           </template>
           <div class="oc-mt" v-else>
             <oc-loader :aria-label="$gettext('Loading personal settings')" />
-            <oc-alert :aria-hidden="true" varition="primary" no-close>
+            <oc-alert :aria-hidden="true" variation="primary" no-close>
```

Comment:
??

partition?

Reply:
Should be fine, this only applies a styling variation on the respective element ;) `variation` is the right prop so thanks for fixing!